crates/cli/src/bin/wasm-bindgen.rs

```diff
@@ -18,7 +18,7 @@ Options:
     --out-dir DIR                Output directory
     --out-name VAR               Set a custom output filename (Without extension. Defaults to crate name)
     --target TARGET              What type of output to generate, valid
-                                 values are [web, bundler, nodejs, no-modules, deno],
+                                 values are [web, bundler, nodejs, no-modules, deno, experimental-nodejs-module],
```

Comment:
Because its experimental its not required, but it would be nice if we add an entry here:
https://github.com/rustwasm/wasm-bindgen/blob/4596d4a633900a3f474c715aa51937ed0282e205/guide/src/reference/deployment.md#L11-L23